### PR TITLE
:sparkles: Cultures routes

### PR DESCRIPTION
**Pull request by Eduardo-Cerqueira**

### CODE REVIEW

File: app/routers/cultures.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    :parameter culture:
+    """
+    db_culture = fetch_culture_by_id(culture_id=culture_id)
+    if not db_culture:
+        raise HTTPException(
+            status_code=404,
+            detail="Culture not found",
+            headers={"X-Error": "Resource not found"},
+        )
+    return update_culture(
```
Comment (ValzM) [inline app/routers/cultures.py]: Need concrete return in JSON format for front-end developers.

File: app/routers/cultures.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    """
+    db_culture = fetch_culture_by_id(culture_id=culture_id)
+    stored_model = CultureOptional(**culture.model_dump())
+    if not db_culture:
+        raise HTTPException(
+            status_code=404,
+            detail="Culture not found",
+            headers={"X-Error": "Resource not found"},
+        )
+    return partial_update_culture(
```
Comment (ValzM) [inline app/routers/cultures.py]: Need concrete return in JSON format for front-end developers.

File: app/routers/cultures.py [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+    :parameter culture_id:
+    """
+    db_culture = fetch_culture_by_id(culture_id=culture_id)
+    if not db_culture:
+        raise HTTPException(
+            status_code=404,
+            detail="Culture not found",
+            headers={"X-Error": "Resource not found"},
+        )
+    return delete_culture(culture_id=culture_id)
```
Comment (ValzM) [inline app/routers/cultures.py]: Need concrete return in JSON format for front-end developers.

File: app/routers/cultures.py (hunk 1 above)
Comment (Eduardo-Cerqueira) [reply app/routers/cultures.py]: Example ?

**Review by ValzM — APPROVED** [review]
✅ LGTM

**Pull request merged**